app/scripts/controllers/transactions/index.js

```diff
@@ -378,14 +378,42 @@ export default class TransactionController extends EventEmitter {
    * updates the params that are editible in the send edit flow
    *
    * @param {string} txId - transaction id
-   * @param {object} editableParams - holds the editable parameters
+   * @param {object} previousGasParams - holds the parameter to update
+   * @param {string} previousGasParams.maxFeePerGas
+   * @param {string} previousGasParams.maxPriorityFeePerGas
+   * @param {string} previousGasParams.gasLimit
+   */
+  updatePreviousGasParams(
```

Comment:
This method should be updated to reflect the changes I made in the other methods: returning the full updated txMeta